src/app/login/login-form.component.html

```diff
@@ -1,18 +1,20 @@
 <form (ngSubmit)="onSubmit()" class="login-form" [formGroup]="userForm">
-  <mat-form-field class="example-full-width">
-    <input matInput i18n-placeholder placeholder="Username" name="name" formControlName = "name" planetLowercase/>
+  <mat-form-field class="full-width">
+    <input matInput i18n-placeholder placeholder="Username" name="name" formControlName = "name" planetLowercase autofocus/>
     <mat-error>
       <planet-form-error-messages [control]="userForm.controls.name"></planet-form-error-messages>
     </mat-error>
   </mat-form-field>
-  <mat-form-field class="example-full-width">
-    <input matInput i18n-placeholder placeholder="Password" name="password" type="password" formControlName = "password"/>
+  <mat-form-field class="full-width">
+    <input matInput i18n-placeholder placeholder="Password" name="password" [type]="showPassword ? 'text' : 'password'" formControlName = "password"/>
+    <mat-icon matSuffix (click)="showPassword = !showPassword" style="cursor: pointer">{{showPassword ? 'visibility' : 'visibility_off'}}</mat-icon>

```

Comment:
@svlesiv Better to have this as a button & mat-icon-button than styling.

Also we avoid inline styling when possible.  For future reference, there's a specific `cursor-pointer` class since this situation is relatively common.

I'd like to merge this, so I'll just update with the button.